src/FHDAlogger.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import datetime
+
+def initiateLogger(origin, level):
+    '''Initiate a logger with given logging level and logging origin
+
+    Args:
+        origin: from which source this logger is from (for determining logging file name)
+        level: the level of the logger, options are [INFO, DEBUG, WARNING, ERROR, CRITICAL]
+    Raises:
+        KeyError: The level parameter is not valid
+    Returns:
+        the logger object
+
+    '''
+    level = level.upper()
+    if level == 'INFO':
```

Comment:
If we pass level directly to `basicConfig`, I think it also works, but will get exception